SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -365,16 +366,18 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	UFUNCTION(BlueprintPure, Category = "Spatial Functional Test")
 	bool WasLoadedFromTakenSnapshot();
 
-	template <typename T>
-	static int GetNumberOfActorsOfType(UWorld* World)
+	template <typename ActorType>
+	static int32 CountActors(UWorld* World)
```

Comment:
This looks like worse performance, why change it?

Reply:
Yeah its a good point I though I wanted a predicated count but it turns out I didn't so this is left over.